src/app/components/SelectNetwork/SelectNetwork.tsx

```diff
@@ -15,7 +15,7 @@ type SelectNetworkProps = {
 	networks: Network[];
 	placeholder?: string;
 	name?: string;
-	value?: string;
+	value?: any;
```

Comment:
Why was this changed to `any`?

Reply:
I pass in the Network object itself, not a string

Reply:
Shouldn't it then be changed to `object`, `Record<string, unknown>` or some specific object?

Reply:
🤔 I'll see how it's used elsewhere